proto/buf/alpha/registry/v1alpha1/user.proto

```diff
@@ -53,6 +54,19 @@ message OrganizationUser {
   OrganizationRoleSource organization_role_source = 4;
 }
 
+message UserSelectedPlugin {
```

Comment:
Is there an existing type for this? What is a user-selected plugin? Needs docs explaining what this is at the minimum

Reply:
we can actually dont need this type anymore. Removed.